src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/ThrottlingRerouteSpec.groovy

```diff
@@ -207,9 +219,18 @@ class ThrottlingRerouteSpec extends HealthCheckSpecification {
         }
         assert brokenIsl, "This should not be possible. Trying to switch port on ISL which is not present in config?"
         antiflap.portDown(sw, port)
-        Wrappers.wait(WAIT_OFFSET, 0) {
-            assert northbound.getLink(brokenIsl).state == IslChangeType.FAILED
+        if (withWait) {
+            Wrappers.wait(WAIT_OFFSET, 0) {
+                assert northbound.getLink(brokenIsl).state == IslChangeType.FAILED
+            }
         }
         return brokenIsl
     }
+
+    Long convertStringTimestampIsoToLong(String timestampIso) {
```

Comment:
I'd keep this as a static method in an Object/Interface like DateTimeConverter.

Reply:
Done, made the method static, and moved to the FlowHelper. Currently we are trying to get rid of such helpers however I could not find where I can put such static method (technically it is related to the flow timestamp). However, if you prefer, I can create a separate class such as DateTimeConverter,

Reply:
> Done, made the method static, and moved to the FlowHelper. Currently we are trying to get rid of such helpers however I could not find where I can put such static method (technically it is related to the flow timestamp). However, if you prefer, I can create a separate class such as DateTimeConverter,

FlowHelper is ok, anyway we're planning to replace it with FlowExtended.